apps/explorer/.env.testnet

```diff
@@ -1,11 +1,12 @@
 # App configuration variables
 NX_CHAIN_EXPLORER_URL=https://explorer.vega.trading/.netlify/functions/chain-explorer-api
-NX_TENDERMINT_URL=https://be.testnet.vega.xyz
+NX_TENDERMINT_URL=https://tm.be.testnet.vega.xyz
 NX_BLOCK_EXPLORER=https://be.testnet.vega.xyz/rest
-NX_TENDERMINT_WEBSOCKET_URL=wss://be.testnet.vega.xyz/
+NX_TENDERMINT_WEBSOCKET_URL=wss://tm.be.testnet.vega.xyz/
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/testnet-network.json
 NX_VEGA_NETWORKS={\"MAINNET"\:\"https://explorer.vega.xyz"\,\"TESTNET\":\"https://explorer.fairground.wtf\"}
 NX_VEGA_ENV=TESTNET
 NX_GITHUB_FEEDBACK_URL=https://github.com/vegaprotocol/feedback/discussions
-NX_VEGA_URL=https://api.n09.testnet.vega.xyz/graphql
+NX_VEGA_URL=https://api.n07.testnet.vega.xyz/graphql
```

Comment:
suggestion (non-blocking): actually, i think you can remove the NX_VEGA_URLs from the env files, that would result in the node switcher finding you a healthy node to connect to, which results in a smoother experience when nodes are down. i think console did this recently.

Reply:
👍 Removed